packages/@lwc/perf-benchmarks/scripts/build.js

```diff
@@ -12,8 +12,8 @@
 const path = require('path');
```

Comment:
These files are part of https://github.com/salesforce/lwc/pull/3484. Once that PR is merged, this PR will be rebased on top of the new master branch.